src/store/actions.ts

```diff
@@ -0,0 +1,82 @@
+// @ts-ignore
+import api from '@molgenis/molgenis-api-client'
+
+export default {
```

Comment:
actions is a really nice place to add some ( more) typing as the model gets constructed ( in part) here.